resources/views/components/input/password/password-strength.blade.php

```diff
@@ -0,0 +1,42 @@
+@slots(['label'])
+
+<div {{ $attributes->only('class')->class('') }}>
+    <x-rapidez::label class="mb-2 text-inactive">
+        {{ $label }}
+    </x-rapidez::label>
+    <password-strength
+        v-slot="passwordStrength"
+        v-bind:min-length="{{ Rapidez::config('customer/password/minimum_password_length', 8) }}"
+        v-bind:min-classes="{{ Rapidez::config('customer/password/required_character_classes_number', 3) }}"
+        {{ $attributes }}
+    >
+        <div class="relative rounded-lg sm:border sm:p-6 sm:shadow lg:p-8">
+            <p v-if="passwordStrength.minClasses < 4" class="text-xs text-inactive mb-2">
+                @lang('Password must have :minClasses differrent characters', ['minClasses' => "@{{ passwordStrength.minClasses }}" ])
+            </p>
+            <div class="mb-4 h-2.5 w-full rounded-full bg-gray-200">
+                <div
+                    v-bind:style="`width:${passwordStrength.strengths.length / 5 * 100}%`"
+                    v-bind:class="passwordStrength.errors.length > 0 ? 'bg-red-500' : 'bg-green-700 !w-full'"
+                    class="h-2.5 rounded-full transition-all duration-300"
+                ></div>
+            </div>
+            <div v-for="error in passwordStrength.errors" class="my-1 flex items-center gap-x-2">
+                <div class="flex size-4 shrink-0 items-center justify-center rounded-full bg-red-500 text-white">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" fill="currentColor" class="size-4 text-red-500">
+                        <path fill-rule="evenodd" d="M8 15A7 7 0 1 0 8 1a7 7 0 0 0 0 14Zm2.78-4.22a.75.75 0 0 1-1.06 0L8 9.06l-1.72 1.72a.75.75 0 1 1-1.06-1.06L6.94 8 5.22 6.28a.75.75 0 0 1 1.06-1.06L8 6.94l1.72-1.72a.75.75 0 1 1 1.06 1.06L9.06 8l1.72 1.72a.75.75 0 0 1 0 1.06Z" clip-rule="evenodd"/>
+                    </svg>
+                </div>
+                <p class="text-sm text-red-500"> @{{ error }}</p>
```

Comment:
Right now if this was used within a form it would not prevent form submission.
A small trick we could use is adding a hidden input with custom validity:
```suggestion
                <p class="text-sm text-red-500 relative">@{{ error }}  <input type="checkbox" oninvalid="this.setCustomValidity(error)" required class="absolute inset-0 opacity-0 pointer-events-none"></p>
```

This would result in a browser message when attempting to submit 
![image](https://github.com/user-attachments/assets/f810c48c-0553-4e8b-9755-ba17d675aeb6)

instead of getting the popup message back from the server after it has gotten submitted.